confd/templates/docker-compose/docker-compose.tmpl

```diff
@@ -489,7 +489,7 @@ services:
       context: docker
       dockerfile: northbound/Dockerfile
     ports:
-      - "8787:8080"
+      - "8080:8080"
```

Comment:
You have same value for "northbound" service (line 467). I will collide.

Reply:
yes, but it won't this is made intentionally to replicate production usecase, check makefile templates

Reply:
I saw them. And didn't get why for we keep northbound, if it is "overriden" by northbound-blue-green ?